paasta_tools/kubernetes_tools.py

```diff
@@ -398,7 +398,9 @@ def get_autoscaling_metric_spec(
         # TODO support multiple metrics
         metrics = []
         target = autoscaling_params["setpoint"] * 100
+        annotations: Dict[str, str] = {}
         # TODO support bespoke PAASTA-15680
+        selector = V1LabelSelector(match_labels={"kubernetes_cluster": cluster})
```

Comment:
This selector affects the signalfx query? It looks like a k8s selector.

Reply:
The sfx adapter reads the selector and converts it dimension
https://github.com/signalfx/signalfx-k8s-metrics-adapter/tree/a3e1f4e374bfe90676e28718a2df7f7fc4e46dd5#signalfx-smart-agent-dependency

HPA queries the metrics server (SFX adapter in this case), so it doesn't care the selector.